easybuild/easyconfigs/i/intelcuda/intelcuda-2017b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Toolchain'
+
+name = 'intelcuda'
+version = '2017b'
+
+homepage = '(none)'
+description = """Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,
+ Intel MPI & Intel MKL, with CUDA toolkit"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2017.4.196'
+gccver = '6.4.0'
+binutilsver = '2.28'
+gccsuff = '-GCC-%s-%s' % (gccver, binutilsver)
+intelver = '%s%s' % (compver, gccsuff)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '-GCCcore-%s' % gccver),
+    ('icc', compver, gccsuff),
+    ('ifort', compver, gccsuff),
+    ('CUDA', '9.0.176', '', ('iccifort', intelver)),
+    ('impi', '2017.3.196', '', ('iccifortcuda', intelver)),
+    ('imkl', '2017.3.196', '', ('iimpic', version)),
+]
```

Comment:
Any particular reason you did it this way instead of how it looks in intelcuda-2016.10.eb ?

Reply:
I simply followed the rule of encapsulating whatever gets repeated into a variable after consulting with @boegel

Reply:
Yeah, but why use
('GCCcore', gccver),
('binutils', binutilsver, ...)
instead of
('iccifort', intelver),

as in intelcuda-2016.10.eb

Reply:
@akesandgren The reason why it is different from `intelcuda-2016.10.eb` is that I took `intel-2017b.eb` as template. I honestly do not know the reason why we prefer to call the separate components instead of the lower toolchain modules. @boegel definitely has better memory than me on that.

That said, I'm fine using the convention you used for the previous `intelcuda`.

Reply:
Ok, I used iccifort in the old one since it made more sense to me at the time, but the GCCcore+binutils like intel-201... uses also makes sense, and keeping it similar to intel.eb makes a lot of sense.

Keep it.

Reply:
There's actually no point in also including `GCCcore` and `binutils` in the `dependencies` list for `intel*` toolchains, other than to clarify that `GCCcore` + `binutils` is used as a base. There's no technical reason for it, everything will work just fine without it.